Patches/python.diff

```diff
@@ -211,50 +205,9 @@ diff -r 8667c26e2bec Objects/codeobject.c
  /* Use co_lnotab to compute the line number from a bytecode index, addrq.  See
     lnotab_notes.txt for the details of the lnotab representation.
  */
-diff -r 8667c26e2bec Objects/object.c
---- a/Objects/object.c	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Objects/object.c	Mon Aug 31 14:44:51 2015 -0700
-@@ -1787,12 +1787,12 @@
- _Py_PrintReferences(FILE *fp)
- {
-     PyObject *op;
--    fprintf(fp, "Remaining objects:\n");
-+    fprintf(fp, "Remaining objects:\r\n");
```

Comment:
If this is an actual issue about line endings then we might want to come up with a permanent solution like a macro that provides the proper line ending for the platform that you can somehow append to a string literal.